src/ray/gcs/gcs_client/test/service_based_gcs_client_test.cc

```diff
@@ -699,6 +699,7 @@ TEST_F(ServiceBasedGcsClientTest, TestNodeHeartbeat) {
   ClientID node_id = ClientID::FromBinary(node_info->node_id());
   auto heartbeat = std::make_shared<rpc::HeartbeatTableData>();
   heartbeat->set_client_id(node_id.Binary());
+  heartbeat->set_should_global_gc(true);
```

Comment:
Why do you have this?

Reply:
If heartbeat did not set `should_global_gc` or contain any resource contents, gcs would accept it but not broadcast(blocked by this `if`):

```
if (heartbeat_data.should_global_gc() ||
      heartbeat_data.resources_available_label_size() > 0 ||
      heartbeat_data.resources_total_label_size() > 0 ||
      heartbeat_data.resource_load_label_size() > 0) {
    heartbeat_buffer_[node_id] = heartbeat_data;
```

Reply:
Got it. Can you write a short comment to explain that? Something like
`Set this flag to broadcast heartbeat because GCS won't publish unchanged heartbeat.`

Reply:
sure